docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -105,105 +112,95 @@ significant changes in each minor release.
 
 
-Release candidates
-^^^^^^^^^^^^^^^^^^^
+.. _development_and_stable_version_maintenance_workflow:
 
-Before a new release or version of Ansible or ``ansible-core`` can be
-done, it will typically go through a release candidate process.
+Development and stable version maintenance workflow
+----------------------------------------------------
 
-This provides the Ansible community the opportunity to test these releases and report
-bugs or issues they might come across.
+In between releases, the Ansible community develops and maintains ``ansible-core`` on GitHub_, with a workflow that looks like this:
 
-Ansible and ``ansible-core`` tag the first release candidate (``RC1``)
-which is usually scheduled to last five business days. The final release
-is done if no major bugs or issues are identified during this period.
+ * Developers add new features and bug fixes to the ``devel`` branch.
+ * Each new feature and each bug fix includes a changelog fragment describing the work.
+ * The development team backports bug fixes to one, two, or three stable branches, depending on the severity of the bug. They do not backport new features.
+ * Release engineers create a minor release for each maintained version every three weeks to ensure that the latest bug fixes are available to users.
+ * At the end of the development period, the release engineers impose a feature freeze and the work of creating a new release begins.
 
-If there are major problems with the first candidate, a second candidate will
-be tagged (``RC2``) once the necessary fixes have landed.
-This second candidate lasts for a shorter duration than the first.
-If no problems have been reported after two business days, the final release is
-done.
+We generally do not provide fixes for unmaintained releases of Ansible, however, there can sometimes be exceptions for critical issues.
 
-More release candidates can be tagged as required, so long as there are
-bugs that the Ansible  or ``ansible-core`` core maintainers consider
-should be fixed before the final release.
+.. _GitHub: https://github.com/ansible/ansible
 
-.. _release_freezing:
+.. _release_changelogs:
 
-Feature freeze
-^^^^^^^^^^^^^^^
+Changelogs
+^^^^^^^^^^
 
-While there is a pending release candidate, the focus of core developers and
-maintainers will on fixes towards the release candidate.
+We generate changelogs based on fragments. When creating new features for existing modules and plugins or fixing bugs, create a changelog fragment describing the change. A changelog entry is not needed for new modules or plugins. Details for those items will be generated from the module documentation.
 
-Merging new features or fixes that are not related to the release candidate may
-be delayed in order to allow the new release to be shipped as soon as possible.
+For guidance on creating changelog fragments, see the :ref:`changelog examples and instructions<changelogs_how_to>` in the Community Guide.
```

Comment:
I don't think all collections in Ansible use our changelog generator.

Reply:
I added a link to the antsibull-changelog tool with a note that we recommend it. We can add more detail in that section later.